pkgs/tools/misc/idevicerestore/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv, fetchFromGitHub, autoreconfHook, pkgconfig
+, curl
+, libimobiledevice
+, libirecovery
+, libzip
+, libusbmuxd
+}:
+
+stdenv.mkDerivation rec {
+  pname = "idevicerestore";
+  version = "2019-02-14";
+
+  src = fetchFromGitHub {
+    owner = "libimobiledevice";
+    repo = pname;
+    rev = "8a882038b2b1e022fbd19eaf8bea51006a373c06";
+    sha256 = "17lisl7ll43ixl1zqwchn7jljrdyl2p9q99w30i6qaci71mas37m";
+  };
+
+  nativeBuildInputs = [
+    autoreconfHook
+    pkgconfig
+
+    curl
```

Comment:
curl should probably be a buildInput instead.

Reply:
Thought these were buildInputs because of the whitespace. The one's below this should be as well.
Add `libplist` too.

Reply:
> curl should probably be a buildInput instead.

Oops, done.

> The one's below this should be as well.

Done.